graphql-jpa-query-schema/src/test/java/com/introproventures/graphql/jpa/query/schema/GraphQLExecutorTests.java

```diff
@@ -474,4 +476,30 @@ public void queryForEntitiesWithWithEmbeddedIdWithWhere() {
         assertThat(result.toString()).isEqualTo(expected);
     }
 
+
+    @Test
+    public void distinctFalse() {
+        //given
+        String query = "{ Books(distinct: false) { select { genre } }}";
+
+        //when
+        LinkedHashMap select = (LinkedHashMap)((LinkedHashMap)executor.execute(query).getData()).get("Books");
+        List books = (List)select.get("select");
+
+        org.junit.Assert.assertTrue(books.size() > 2);
+    }
+
+    @Test
+    public void ignoreFilter() {
+        //given
+        String query = "{ Books(where: {title: {EQ: \"title\"}} ) { select { id title } }}";;
+
+        String expected = "[ValidationError{validationErrorType=WrongType, queryPath=[Books], message=Validation error of type WrongType: argument 'where' with value 'ObjectValue{objectFields=[ObjectField{name='title', value=ObjectValue{objectFields=[ObjectField{name='EQ', value=StringValue{value='title'}}]}}]}' contains a field not in 'BooksCriteriaExpression': 'title' @ 'Books', locations=[SourceLocation{line=1, column=9}], description='argument 'where' with value 'ObjectValue{objectFields=[ObjectField{name='title', value=ObjectValue{objectFields=[ObjectField{name='EQ', value=StringValue{value='title'}}]}}]}' contains a field not in 'BooksCriteriaExpression': 'title''}]";
+
+        //when
+        Object result = executor.execute(query).getErrors();
+
+        // then
+        assertThat(result.toString()).isEqualTo(expected);
```

Comment:
@miklemv Let's use the following assertion method for errors: https://github.com/introproventures/graphql-jpa-query/blob/2933500644bd6b781919a24c5583c6708f046a13/graphql-jpa-query-schema/src/test/java/com/introproventures/graphql/jpa/query/schema/CalculatedEntityTests.java#L79